### ISSUE TITLE
Clarification of `observed data`

### ISSUE THREAD

**Issue by brandomr** [uncharted-causemos/docs#5]
Is the idea behind `observed data` that there would be some datasets that a user uploads which represent ground truth that could be used for assessing the model?

For example, if the model predicts crop production, then a `observed data` would be some set of data cubes which represent "actuals" for crop production (e.g. from the Central Statistical Agency etc)?

**Comment by mkozlowski-uncharted** [comment]
Correct, this field arose from our discussions on tracking provenance. A model could include some data that would be exposed as "cubes" in Causemos that the user could use for performing transforms or comparative analysis.

I added it as an array in the model structure because presumably these cubes would be available and known right away during model registration. We could also move this to be a flag on an actual cube if it makes it easier.
